contrib/macOS/resources/English.lproj/License.txt

```diff
@@ -0,0 +1,828 @@
+
+OpenZFS on OS X consists of software that is covered under the
+Common Development and Distribution License 1.0 and the Apple
+Public Source License 2.0.
+
+Each file at https://github.com/openzfsonosx records which license it is
+released under.
+
+Much of OpenZFS on OS X is a port of ZFS on Linux.
+
+--------------------------------------------------------------------
+ZFS on Linux COPYRIGHT
```

Comment:
many ZoL references... 
While we are actually working actively on removing those, in favor of OpenZFS

Reply:
Ah these pkg files haven't been looked at since the start - could do with a refresher.